### PR TITLE
feat : Added Thai language

### PR DESCRIPTION
**Pull request by watchakorn-18k**

### CODE REVIEW

File: frontend/src/lang/th.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    "PasswordsDoNotMatch": "รหัสผ่านไม่ตรงกัน",
+    "Repeat Password": "ยืนยันรหัสผ่าน",
+    "Create": "สร้าง",
+    "signedInDisp": "ลงชื่อเข้าใช้ในชื่อ {0}",
+    "signedInDispDisabled": "ปิดใช้งาน Auth",
+    "home": "หน้าหลักe",
+    "console": "คอนโซล",
+    "registry": "Registry",
+    "compose": "Compose",
+    "addFirstStackMsg": "Compose stack แรกของคุณ",
```
Comment (Zaid-maker) [inline frontend/src/lang/th.json:14]: looks like the translations is done completly

File: frontend/src/lang/th.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    "authIncorrectCreds": "ชื่อผู้ใช้หรือรหัสผ่านไม่ถูกต้อง",
+    "PasswordsDoNotMatch": "รหัสผ่านไม่ตรงกัน",
+    "Repeat Password": "ยืนยันรหัสผ่าน",
+    "Create": "สร้าง",
+    "signedInDisp": "ลงชื่อเข้าใช้ในชื่อ {0}",
+    "signedInDispDisabled": "ปิดใช้งาน Auth",
+    "home": "หน้าหลักe",
+    "console": "คอนโซล",
+    "registry": "Registry",
+    "compose": "Compose",
```
Comment (Zaid-maker) [inline frontend/src/lang/th.json:13]: here too

File: frontend/src/lang/th.json [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    "Create your admin account": "สร้างบัญชีผู้ดูแลระบบของคุณ",
+    "authIncorrectCreds": "ชื่อผู้ใช้หรือรหัสผ่านไม่ถูกต้อง",
+    "PasswordsDoNotMatch": "รหัสผ่านไม่ตรงกัน",
+    "Repeat Password": "ยืนยันรหัสผ่าน",
+    "Create": "สร้าง",
+    "signedInDisp": "ลงชื่อเข้าใช้ในชื่อ {0}",
+    "signedInDispDisabled": "ปิดใช้งาน Auth",
+    "home": "หน้าหลักe",
+    "console": "คอนโซล",
+    "registry": "Registry",
```
Comment (Zaid-maker) [inline frontend/src/lang/th.json:12]: here aswell

**Review by Zaid-maker — CHANGES_REQUESTED** [review]

File: frontend/src/lang/th.json (hunk 1 above)
Comment (watchakorn-18k) [reply frontend/src/lang/th.json:14]: I purposely used the English transliteration because Thai people are very used to this.

Comment (Zaid-maker) [reply frontend/src/lang/th.json:14]: > I purposely used the English transliteration because Thai people are very used to this.

ohhh ok! Thanks

**Pull request merged**